### PR TITLE
[ArchiveFilesV2] Migrated to Node 10

### PR DESCRIPTION
**Pull request by egor-bryzgalov**
**Task name**: ArchiveFilesV2

**Description**: Migrated task to Node 10.

**Documentation changes required:** No

**Added unit tests:** No

**Attached related issue:** [#433](https://github.com/microsoft/build-task-team/issues/433)

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by DaniilShmelev — APPROVED** [review]

File: Tasks/ArchiveFilesV2/Tests/L0.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ describe('ArchiveFiles L0 Suite', function () {
     
     tl.setResourcePath(path.join( __dirname, '..', 'task.json'));
     cases.forEach(function(numberOfFiles) {
-        it('Verify plan output for ' + numberOfFiles + ' files has correct number of lines', (done: MochaDone) => {
+        it('Verify plan output for ' + numberOfFiles + ' files has correct number of lines', (done: Mocha.Done) => {
```
Comment (alexander-smolyakov) [inline Tasks/ArchiveFilesV2/Tests/L0.ts:44]: Nit: Could you replace strings contention by template literal?

**Review by alexander-smolyakov — APPROVED** [review]
LGTM, but also please take a look at the comment.
Thanks!

File: Tasks/ArchiveFilesV2/Tests/L0.ts (hunk 1 above)
Comment (egor-bryzgalov) [reply Tasks/ArchiveFilesV2/Tests/L0.ts]: Thanks! Updated

**Comment by alex-peck** [comment]
Can you also delete the entries in the Tests-Legacy folder related to ArchiveFiles?

**Review by anatolybolshakov — APPROVED** [review]

**Review by EzzhevNikita — APPROVED** [review]

**Comment by egor-bryzgalov** [comment]
> Can you also delete the entries in the Tests-Legacy folder related to ArchiveFiles?

Thank you! I've deleted them.

**Review by alex-peck — APPROVED** [review]

**Pull request merged**